fastlane/lib/assets/completions/completion.bash

```diff
@@ -0,0 +1,19 @@
+_fastlane_complete() {
+  COMPREPLY=()
+  local word="${COMP_WORDS[COMP_CWORD]}"
+  local completions=""
+
+  # look for Fastfile either in this directory or fastlane/ then grab the lane names
+  if [[ -e "Fastfile" ]]; then
+    file="Fastfile"
+  elif [[ -e "fastlane/Fastfile" ]]; then
+    file="fastlane/Fastfile"
+  fi
+
+  # parse 'beta' out of 'lane :beta do', etc
+  completions=`cat $file | grep "lane \:" | awk -F ':' '{print $2}' | awk -F ' ' '{print $1}'`
```

Comment:
It will also not work with lanes that are created in a loop or similar, but that's ok for now, because it's blazing fast :rocket: